tools/clang/test/CodeGenSPIRV/texture.load.vulkan1.2.hlsl

```diff
@@ -0,0 +1,160 @@
+// Run: %dxc -T ps_6_0 -E main -fspv-target-env=vulkan1.2
```

Comment:
The main change in this PR is about OpEntryPoint. So the only CHECK that's really necessary are the first two ones:
```
CHECK: OpEntryPoint Fragment %main "main"
CHECK-SAME: %t1 %t2 %t3 %t4 %t5 %t6 %t7 %t8 %in_var_A %in_var_B %out_var_SV_Target
```

The rest of the CHECKs are not relevant to the Vulkan environment, so there is no reason to include them. So you can actually remove most of the `main` function body.

Also here we only have resources (textures). We should also add other kind of globals to the shader. For example standalone globals variables and cbuffers (see `var.globals.hlsl` test as an example).